### ISSUE TITLE
Cannot preset date only and also not use format DD.MM.YYYY

### ISSUE THREAD

**Issue by FROGGS** [xdan/periodpicker#16]
When I do this:

``` html
<input type="text" value="12.04.2016" id="datetimepicker"/>
<script>
jQuery('#datetimepicker').periodpicker({
    timepicker: false,
    formatDate: 'DD.MM.YYYY'
});
</script>
```

I get "`1 January 2016`".

And when I add the time to the field which I basically dont want:

``` html
<input type="text" value="00:00 12.04.2016" id="datetimepicker"/>
<script>
jQuery('#datetimepicker').periodpicker({
    timepicker: false,
    formatDate: 'DD.MM.YYYY'
});
</script>
```

I get "`20 April 2012`", so one could guess that it still tries to parse it as YY/MM/DD.

How can I circumvent both problems?

**Comment by xdan** [comment]
Hi, update here http://xdsoft.net/jqplugins/periodpicker/?update#plugin_sender please
Fixed in 4.0.0

12.04.2016, 19:39, "Tobias Leich" notifications@github.com:

> When I do this:
> 
> <input type="text" value="12.04.2016" id="datetimepicker"/> <script> jQuery('#datetimepicker').periodpicker({ timepicker: false, formatDate: 'DD.MM.YYYY' }); </script>
> 
> I get "1 January 2016".
> 
> And when I add the time to the field which I basically dont want:
> 
> <input type="text" value="00:00 12.04.2016" id="datetimepicker"/> <script> jQuery('#datetimepicker').periodpicker({ timepicker: false, formatDate: 'DD.MM.YYYY' }); </script>
> 
> I get "20 April 2012", so one could guess that it still tries to parse it as YY/MM/DD.
> 
> How can I circumvent both problems?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub

## 

C уважением,
Чупурнов Валерий
http://xdan.ru

**Comment by FROGGS** [comment]
Hi, I can confirm that version 4.0.0 fixes this issue. (Had 3.4.1)

Though, in the index.html example the jsvscript source file should be:
build/jquery.periodpicker.full.min.js
